docs/source/utils/notebook.py

```diff
@@ -14,6 +14,13 @@ def inner(example_text: str):
 
 # %%
 !python3 -m pip install -q dff[examples]
+# Installs dff with dependencies for running examples
+# To install the minimal version of dff, use `pip install dff`
+# To install other options of dff, use `pip install dff[OPTION_NAME1,OPTION_NAME2]`
+# where OPTION_NAME can be one of the options from EXTRA_DEPENDENCIES.
+# e.g `pip install dff[ydb, mysql]` installs dff with dependencies for using Yandex Database and MySQL
```

Comment:
I think it's better to show how to install only one dependency; and then show how to install multiple:
```suggestion
# To install extra dependencies for dff, use `pip install dff[OPTION_NAME]`
# where OPTION_NAME can be one of the options from EXTRA_DEPENDENCIES.
# e.g `pip install dff[mysql]` installs dff with dependencies for using MySQL
# To install multiple dependencies at once, use `pip install dff[OPTION_NAME1,OPTION_NAME2, ...]`
```